### ISSUE TITLE
windows build script

### ISSUE THREAD

**Issue by berndhahnebach** [IfcOpenShell/IfcOpenShell#155]
Just tried the windows build script after a bread of a few weeks and got an error at boost ...

`
Extracting  boost_1_59_0\boost\accumulators\framework\parameters
Extracting  boost_1_59_0\boost\accumulators\framework\accumulators
Extracting  boost_1_59_0\boost\accumulators\framework
Extracting  boost_1_59_0\boost\accumulators
Extracting  boost_1_59_0\boost
Extracting  boost_1_59_0

Everything is Ok

Folders: 4351
Files: 47028
Size:       436235290
Compressed: 65488392
Zugriff verweigert
Building RelWithDebInfo Boost 1.59.0. Please be patient this will take a while.
MSBUILD : error MSB1009: Die Projektdatei ist nicht vorhanden.
Schalter: C:\Users\bhb\Desktop\ifcos\ifcosdev\deps\boost\build-vs2013-x64\OCE.sln

An error occurred

Build ended at  9:32:01.09. Time elapsed 0:02:53.17.

C:\Users\bhb\Desktop\ifcos\ifcosdev\win>
`

Ahh:
Zugriff verweigert --> access denied
Die Projektdatei ist nicht vorhanden.  --> the project file was not found

**Comment by aothms** [comment]
@Stinkfist0 did you happen to have a look at this? Why would there be an `OCE.sln` in the `deps\boost\` directory? @berndhahnebach did you happen to make any progress with this?

**Comment by Stinkfist0** [comment]
This sounds rather strange as the build script has been rather stable for quite a while. I'd recommend simply rerunning the script. If the problem persists, I'd try first removing the boost folder and retrying. Same for OCE if rebuilding Boost didn't help.

**Comment by berndhahnebach** [comment]
> I'd recommend simply rerunning the script.
I did rerun the script before I made the first post and got the same error. Than I made the post. 

Just did a fresh rerun again and it passed this without a problem ... Means thes issue could be closed. 

Thanks guys and sorry for bothering you in this regard ...

At the moment it compiles collada. It will take a while until all is compiled. 

Bernd
